LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -232,7 +237,7 @@ function initiateWidget(serviceID) {
                                 cache: false
                             });
                         }
-                        getMembers(serviceID);
+                        //getMembers(serviceID);
```

Comment:
This is duplicate code so should be removed.

Reply:
Removed